corehq/apps/userreports/models.py

```diff
@@ -778,7 +785,53 @@ def ui_filters(self):
     @property
     @memoized
     def charts(self):
-        return [ChartFactory.from_spec(g._obj) for g in self.configured_charts]
+        if (
+            self.config_id and self.configured_charts
+            and toggles.SUPPORT_EXPANDED_COLUMN_IN_REPORTS.enabled(self.domain)
```

Comment:
It looks like this toggle only affects the backend, not the UI? Does that mean these options are selectable but broken for all other users?

Reply:
> It looks like this toggle only affects the backend, not the UI?

The UI uses this as well
https://github.com/dimagi/commcare-hq/blob/97410667267b6cbe329503f59ee71751ad194f9c/corehq/apps/userreports/reports/view.py#L586. So now they would have the display information about the expanded columns. The view already had the data but not the display information which this change now provides (more in the PR description)

> Does that mean these options are selectable but broken for all other users?

The charts show an empty space for the user instead of a bar line on multibars, for example if you see the bar graph here https://www.commcarehq.org/a/gcc-sangath-dev/reports/builder/edit/7c76e6ddbc5ad7f7b679089e482d1fc9/ you would see empty spaces.

HQ is explicit that it does not support that column type 
https://github.com/dimagi/commcare-hq/pull/27345/files#diff-a3a617fe0d1aece479df5cb8ab2505d4dcb1d3ec29475bc262e6c672e4ef614aR241